### ISSUE TITLE
Homepage Tidy Up

### ISSUE THREAD

**Issue by ytsruh** [ytsruh/wrongmove#73]
Homepage currently has a tonne of links with no functionality / broken links.

@RyanBriggsDev in the spirit of trying to get this project over line I think we should do one fo the following:
1) Remove the links that we wont be used
2) Create a new page (feature not available or similar) and link to that.

I'm in favour of Option 2 becuase its a lot easier & preserves your hard work on cloning the Rightmove homepage. What are your thoughts?

**Comment by RyanBriggsDev** [comment]
Yeah, let's go with option 2. Will tidy things up nicely

**Comment by ytsruh** [comment]
Assigning this to myself on the basis that EVEN I have the CSS capabilities to do this......probably

**Comment by ytsruh** [comment]
Changing this to you so you can tidy it up as you feel fit. I've created a new issue (#76) for me to create the 'coming soon' page that you can link to

**Comment by RyanBriggsDev** [comment]
Will tidy up once 'coming soon' page is complete. Do you want me to do this @ytsruh?

**Comment by ytsruh** [comment]
Yeah go for it. Im going to message that Shyam guy again but no point in waiting
